public/lib/helpers/timeline_helper.js

```diff
@@ -125,10 +125,22 @@ export default class TimelineHelper {
     .sort(function (a, b) {
       //same count, return alphabetic order
       if (counts.get(a) === counts.get(b)) {
-        return a > b;
+        if (a > b) {
```

Comment:
Can you set `counts.get(a)` and `counts.get(b)` to variables on e.g. line 126 and use those variables wherever you call the functions?